assets/javascripts/discourse/initializers/discourse-calendar.js

```diff
@@ -328,16 +350,20 @@ function initializeDiscourseCalendar(api) {
       return d.hours() !== 0 || d.minutes() !== 0 || d.seconds() !== 0;
     };
 
+    const hasTime =
+      hasTimeSpecified(to.dateTime) || hasTimeSpecified(from.dateTime);
```

Comment:
This line is causing a JS error locally when I have events without a `to` date. Calendar is empty and I see this: 

<img width="1409" alt="image" src="https://github.com/discourse/discourse-calendar/assets/368961/66378a2b-f533-47dd-8111-eaab161f8489">